src/languageSupport/languages/flux/parser/index.ts

```diff
@@ -1,8 +1,12 @@
 import {File} from 'src/types/ast'
 import {
+  initLog,
   parse as flux_parse,
   format_from_js_file as flux_format_from_js_file,
 } from '@influxdata/flux-lsp-browser'
+
+initLog()
```

Comment:
Since the LSP doesn't have a single entry point, @hoorayimhelping and I agreed to leave this code as the best alternative. Doc'ing the tech debt in the LSP, that we need some refactoring to enable the single entry point.